### ISSUE TITLE
Using tensor to build torchio.Image

### ISSUE THREAD

**Issue by wendy-xiaozong** [fepegar/torchio#184]
I am using tensor to build the torchio.Image like this:
```
        torchio.Subject(
            img=tio.Image(tensor=img, label=tio.INTENSITY),  # image to be segmented
            label=tio.Image(tensor=label, label=tio.LABEL),  # brain mask we are predicting
        )
```
`img, label` are three dimension tensors.
When I am trying to run this code `for batch_idx, batch in enumerate(loader):` 
`loader` is `torch.utils.data.DataLoader` classd
I run into this error:
```
  File "/localscratch/jueqi.32501327.0/env/lib/python3.6/site-packages/torch/utils/data/_utils/worker.py", line 178, in _worker_loop
    data = fetcher.fetch(index)
  File "/localscratch/jueqi.32501327.0/env/lib/python3.6/site-packages/torch/utils/data/_utils/fetch.py", line 47, in fetch
    return self.collate_fn(data)
  File "/localscratch/jueqi.32501327.0/env/lib/python3.6/site-packages/torch/utils/data/_utils/collate.py", line 74, in default_collate
    return {key: default_collate([d[key] for d in batch]) for key in elem}
  File "/localscratch/jueqi.32501327.0/env/lib/python3.6/site-packages/torch/utils/data/_utils/collate.py", line 74, in <dictcomp>
    return {key: default_collate([d[key] for d in batch]) for key in elem}
  File "/localscratch/jueqi.32501327.0/env/lib/python3.6/site-packages/torch/utils/data/_utils/collate.py", line 74, in default_collate
    return {key: default_collate([d[key] for d in batch]) for key in elem}
  File "/localscratch/jueqi.32501327.0/env/lib/python3.6/site-packages/torch/utils/data/_utils/collate.py", line 74, in <dictcomp>
    return {key: default_collate([d[key] for d in batch]) for key in elem}
  File "/localscratch/jueqi.32501327.0/env/lib/python3.6/site-packages/torch/utils/data/_utils/collate.py", line 81, in default_collate
    raise TypeError(default_collate_err_msg_format.format(elem_type))
TypeError: default_collate: batch must contain tensors, numpy arrays, numbers, dicts or lists; found <class 'NoneType'>
```

I'm a newbie to pytorch, just started to write it a few days ago. I do not know what I did wrong. Also. thanks a lot for writing this tool!

**Comment by fepegar** [comment]
Thankfully I just got this error too! I'll commit the fix in a second.

**Comment by fepegar** [comment]
Fixed in `v0.16.21`. Thanks for reporting!

**Comment by wendy-xiaozong** [comment]
Thanks! it is working now!
